Functions/PollEmailProcessing.cs

```diff
@@ -0,0 +1,60 @@
+using System.Text.RegularExpressions;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+
+namespace Functions
+{
+    public class PollEmailProcessing
+    {
+        private readonly ILogger _logger;
+
+        public PollEmailProcessing(ILoggerFactory loggerFactory)
+        {
+            _logger = loggerFactory.CreateLogger<PollEmailProcessing>();
+        }
+
+        [Function("PollEmailProcessing")]
+        public async Task RunAysnc([TimerTrigger("* * * * *")] MyInfo myTimer)
```

Comment:
We should also add `RunOnStartup = true, UseMonitor = true`. Here is the [doc](https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-timer?tabs=python-v2%2Cisolated-process&pivots=programming-language-csharp#:~:text=%25ScheduleAppSetting%25.-,RunOnStartup,-If%20true%2C%20the).
i.e:
  ```
public async Task RunAysnc([TimerTrigger("* * * * *", RunOnStartup = true, 
            UseMonitor = true)] MyInfo myTimer)
```